distributed/worker.py

```diff
@@ -4576,3 +4591,74 @@ def warn(*args, **kwargs):
         worker.log_event("warn", {"args": args, "kwargs": kwargs})
 
     warnings.warn(*args, **kwargs)
+
+
+def benchmark_disk(
+    sizes=["1 kiB", "100 kiB", "1 MiB", "10 MiB", "100 MiB"],
+    rootdir=None,
+    duration=1.0,
+) -> dict:
+    duration = parse_timedelta(duration)
+
+    out = {}
+    for size_str in sizes:
+        with tmpdir(dir=rootdir) as dir:
+            dir = pathlib.Path(dir)
+            names = list(map(str, range(100)))
+            size = parse_bytes(size_str)
+
+            data = os.urandom(size)
+
+            start = time()
+            total = 0
+            while time() < start + duration:
+                with open(dir / random.choice(names), mode="ab") as f:
+                    f.write(data)
+                    os.fsync(f)
```

Comment:
```suggestion
                    f.flush()
                    os.fsync(f.fileno())
```

Reply:
https://docs.python.org/3/library/os.html#os.fsync
> If you’re starting with a buffered Python [file object](https://docs.python.org/3/glossary.html#term-file-object) f, first do f.flush(), and then do os.fsync(f.fileno()), to ensure that all internal buffers associated with f are written to disk.

Reply:
Oh interesting.  Cool.  Thank you for the pointer.